yoti-sdk-spring-boot-example/src/main/resources/templates/profile.html

```diff
@@ -1,88 +1,99 @@
-                <td th:replace="fragments :: anchorDisplay(${phoneNumberVerifiers})" />
-            </tr>
-            <tr th:if="${emailAddress != null}">
-                <td>Email Address:</td>
-                <td><p th:text="${emailAddress}"/></td>
-                <td th:replace="fragments :: anchorDisplay(${emailAddressSources})" />
-                <td th:replace="fragments :: anchorDisplay(${emailAddressVerifiers})" />
-            </tr>
-        </table>
+
+    <body class="yoti-body">
+        <main class="yoti-profile-layout">
+
+            <section class="yoti-profile-user-section">
+                <div class="yoti-profile-picture-powered-section">
+                    <span class="yoti-profile-picture-powered">Powered by</span>
+                    <img class="yoti-logo-image" th:src="@{/static/assets/logo.png}" th:srcset="@{/static/assets/logo@2x.png}" alt="Yoti" />
+                </div>
+
+                <div class="yoti-profile-picture-section">
+
+                    <div class="yoti-profile-picture-area" th:if="${base64Selfie != null}">
+                        <img th:src="${base64Selfie}" class="yoti-profile-picture-image" alt="Yoti" />
+                        <i class="yoti-profile-picture-verified-icon"></i>
+                    </div>
+
+                    <div class="yoti-profile-name" th:if="${fullName != null}">
+                        <span th:text="${fullName}" />
+                    </div>
+
+                </div>
+            </section>
+
+            <section class="yoti-attributes-section">
+
+                <div th:if="${appLogo == null}">
+                    <img class="yoti-company-logo" th:src="@{/static/assets/company-logo.jpg}" alt="company logo" />
+                </div>
+                <div th:if="${appLogo != null}">
+                    <img class="yoti-company-logo" th:src="${appLogo}" alt="company logo" />
+                </div>
+
+                <div class="yoti-attribute-list-header">
+                    <div class="yoti-attribute-list-header-attribute">Attribute</div>
+                    <div class="yoti-attribute-list-header-value">Value</div>
+                    <div>Anchors</div>
+                </div>
+
+                <div class="yoti-attribute-list-subheader">
+                    <div class="yoti-attribute-list-subhead-layout">
+                        <div>S / V</div>
+                        <div>Value</div>
+                        <div>Sub type</div>
+                    </div>
+                </div>
+
+                <div class="yoti-attribute-list">
+
+                    <div th:each="attribute : ${displayAttributes}" class="yoti-attribute-list-item">
+
+                        <div class="yoti-attribute-name">
+                            <div class="yoti-attribute-name-cell">
+                                <i th:class="${attribute.icon}" />
+                                <span class="yoti-attribute-name-cell-text" th:text="${attribute.displayName}" />
```

Comment:
Will this work if the attribute is not ticked in the Dashboard' scenario tab?

Reply:
This will only try to display attributes that were returned in the profile.